bin/v2.js

```diff
@@ -525,8 +555,111 @@ const releaseCmd = (cmdObj, bucketNames, lambdaFunctions, environments, bucketPa
     .catch(err => console.error(err));
 }
 
+const cleanupCmd = async (cmdObj, bucketNames, lambdaFunctions, environments, bucketPath, packageName) => {
+
+  const now = new Date();
+  const ninetyDaysAgo = new Date(now.getTime() - (1000 * 60 * 60 * 24 * 90));
+
+
+  // This line might be too cleaver. It's getting all the server environments and then getting an array of just the unique ones.
+  // noinspection JSUnresolvedVariable
+  const serverEnvironments = [...new Set(environments.map(environment => environment.serverEnvironment))];
+
+  const confirmationQuestions = serverEnvironments.map(serverEnvironment => {
+    const bucketName = _.get(bucketNames, serverEnvironment);
+    return {
+      type: 'confirm',
+      name: serverEnvironment,
+      message: `Do you want to clean up the gurgler assets in the S3 bucket ${bucketName} with the path: ${bucketPath}?`,
+      default: false
+    }
+  });
+
+  inquirer.prompt(confirmationQuestions).then(answers => {
+    serverEnvironments.map(serverEnvironment => {
+      if (answers[serverEnvironment]) {
+        const bucketName = _.get(bucketNames, serverEnvironment);
+        console.log(`Cleaning up gurgler assets in the S3 bucket ${bucketName} with the path: ${bucketPath} `);
+
+        requestCurrentlyReleasedVersions(environments).then(releasedVersions => {
+          return getDeployedVersionListWithMetadata(bucketName, bucketPath, packageName).then(deployedArtifacts => {
+
+            console.log(`There are currently a total of ${deployedArtifacts.length} deployed artifacts in ${serverEnvironment}.`);
+
+            const oldArtifacts = deployedArtifacts
+              .filter(artifact => {
+                // only allow an asset to be deleted if it's older than a year
+                return artifact.lastModified.getTime() < ninetyDaysAgo.getTime()
+              })
+              .filter(artifact => {
+                // only allow an asset to be deleted if it's not being used in any of the environments
+                for (const releasedVersion of releasedVersions) {
+                  // noinspection JSUnresolvedVariable
+                  if (releasedVersion.releasedHash === artifact.hash) {
+                    return false;
+                  }
+                }
+                return true;
+              })
+            console.log(`We are going to delete ${oldArtifacts.length} ${packageName} artifacts in ${serverEnvironment}`);
```

Comment:
```suggestion
            console.log(`We are going to delete ${oldArtifacts.length} ${packageName} artifact(s) in ${serverEnvironment}`);
```

For that day this only deletes one artifact 😄